scene/resources/default_theme/default_theme.cpp

```diff
@@ -858,6 +858,13 @@ void fill_default_theme(Ref<Theme> &theme, const Ref<Font> &default_font, const
 	theme->set_constant("bezier_len_pos", "GraphEdit", 80 * scale);
 	theme->set_constant("bezier_len_neg", "GraphEdit", 160 * scale);
 
+	// Asset store
+
+	theme->set_icon("RatingNoStar", "EditorIcons", make_icon(star_png));
+	theme->set_icon("RatingStar", "EditorIcons", make_icon(star_active_png));
```

Comment:
not good, deafult_theme should not be touched except for bug fixes, and this is not one, check my comment on how to load icons.